internal/client.go

```diff
@@ -457,6 +457,9 @@ type (
 		// when there are no active RPCs, Time and Timeout will be ignored and no
 		// keepalive pings will be sent.
 		KeepAlivePermitWithoutStream bool
+
+		// MaxPayloadSize is a number of bytes that gRPC would allow to travel to and from server. Defaults to 64 MB.
```

Comment:
server will also do input size validation, so `travel to` is not really necessary
the main point is to allow SDK to successfully replay the workflow history